test/core/end2end/tests/connectivity.cc

```diff
@@ -116,7 +121,7 @@ static void test_connectivity(grpc_end2end_test_config config) {
 
   /* quickly followed by a transition to TRANSIENT_FAILURE */
   grpc_channel_watch_connectivity_state(f.client, GRPC_CHANNEL_CONNECTING,
-                                        grpc_timeout_seconds_to_deadline(3),
+                                        grpc_timeout_seconds_to_deadline(10),
```

Comment:
With respect to the flakiness bug we're solving, this change made no difference in my testing.

Reply:
I increased since we increased the timeout. Anything above 5 should be fine here imo